thinkhazard/views/pdf.py

```diff
@@ -0,0 +1,143 @@
+
+import datetime
+
+from os import path
+from random import randint
+
+from subprocess import (
+    Popen,
+    PIPE,
+)
+
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPBadRequest
+from pyramid.response import FileResponse
+
+from .report import (
+    _hazardlevel_nodata,
+    get_division,
+    get_hazard_types,
+    get_info_for_hazard_type,
+    get_parents,
+)
+
+from ..models import (
+    DBSession,
+    AdministrativeDivision,
+    HazardLevel,
+    HazardCategory,
+    HazardType,
+    HazardCategoryAdministrativeDivisionAssociation,
+)
+
+
+@view_config(
+    route_name='pdf_cover', renderer='templates/pdf_cover.jinja2')
+def report_print(request):
+    try:
+        division_code = request.matchdict.get('divisioncode')
+    except:
+        raise HTTPBadRequest(detail='incorrect value for parameter '
+                                    '"divisioncode"')
+    division = get_division(division_code)
+    hazard_types = get_hazard_types(division_code)
+
+    hazards_sorted = sorted(hazard_types, key=lambda a: a['hazardlevel'].order)
+
+    hazard_categories = []
+    for h in hazards_sorted:
+        if h['hazardlevel'].mnemonic == _hazardlevel_nodata.mnemonic:
+            continue
+        hazard_categories.append(get_info_for_hazard_type(
+            h['hazardtype'].mnemonic, division))
+
+    context = {
+        'hazards': hazard_types,
+        'hazards_sorted': sorted(hazard_types,
+                                 key=lambda a: a['hazardlevel'].order),
+        'parents': get_parents(division),
+        'division': division,
+        'hazard_categories': hazard_categories,
+        'date': datetime.datetime.now()
+    }
+
+    return context
+
+
+@view_config(route_name='report_pdf')
+def report_pdf(request):
+    try:
+        division_code = request.matchdict.get('divisioncode')
+    except:
+        raise HTTPBadRequest(detail='incorrect value for parameter '
+                                    '"divisioncode"')
+
+    date = datetime.datetime.now()
+    filename = path.join(
+        request.registry.settings.get('pdf_archive_path'),
+        division_code + '-' + date.strftime('%Y-%m-%d-%H:%M:%S') +
+        '-{:d}'.format(randint(1, 1E6)) + '.pdf')
+
+    categories = DBSession.query(HazardCategory) \
+        .join(HazardCategoryAdministrativeDivisionAssociation) \
+        .join(AdministrativeDivision) \
+        .join(HazardLevel) \
+        .join(HazardType) \
+        .filter(AdministrativeDivision.code == division_code) \
+        .order_by(HazardLevel.order)
+
+    pages = ''
+    for cat in categories:
+        pages = pages + ' page "{}"'.format(
+            request.route_url(
+                'report_print',
+                divisioncode=division_code,
+                hazardtype=cat.hazardtype.mnemonic
+            )
+        )
+
+    cover_url = request.route_url('pdf_cover', divisioncode=division_code)
+
+    command = \
```

Comment:
Here he does not find wkhtmltopdf.
I remember that we had the same problem with developement.ini in processing load settings with wsgi.
Current folder is not project root. Path should be absolute ex:
```
settings = get_appsettings(os.path.join(os.path.dirname(__file__),
                                            '../../development.ini'))
```